projects/demo/src/modules/experimental/pin/examples/1/index.less

```diff
@@ -0,0 +1,14 @@
+:host {
+    display: flex;
+    gap: 2rem;
+    background: var(--tui-base-02);
+    box-shadow: 0 0 0 2rem var(--tui-base-02);
+    padding: 1rem 0 0 1rem;
+    margin-bottom: -1rem;
```

Comment:
It looks strange 🤔 

Example should not have any extra space below content.
Maybe something goes wrong ?

Reply:
Is it connected to `transform: translate(-50%, -50%)` (on `<tui-pin />`) ?

Reply:
Yes, it is. Pins would most likely be positioned on a map of sorts so we want to position their center at top/left.